FindPOLine/utils.js

```diff
@@ -59,8 +59,8 @@ export const buildOrderLinesQuery = (queryParams, isbnId, normalizedISBN, locale
       [FILTERS.RECEIPT_DUE]: buildDateRangeQuery.bind(null, `physical.${FILTERS.RECEIPT_DUE}`),
       [FILTERS.CLAIM_SENT]: buildDateRangeQuery.bind(null, [FILTERS.CLAIM_SENT]),
       [FILTERS.TAGS]: buildArrayFieldQuery.bind(null, [FILTERS.TAGS]),
-      [FILTERS.FUND_CODE]: (filterValue) => `fundDistribution =/@fundId ${filterValue}`,
-      [FILTERS.EXPENSE_CLASS]: buildArrayFieldQuery.bind(null, ['fundDistribution']),
+      [FILTERS.FUND_CODE]: buildArrayFieldQuery.bind(null, FILTERS.FUND_DISTRIBUTION),
```

Comment:
@alisher-epam It seems that it's a regression https://github.com/folio-org/ui-plugin-find-po-line/pull/138/files